src/js/view/view.js

```diff
@@ -823,6 +824,10 @@ define([
             }
 
             _captionsRenderer.resize();
+            if (_isMobile) {
```

Comment:
I had this check because we enlarged the icon size for mobile and that was causing it to cut off.
But I think you are right that we should still hide the time on small players.
Updated the PR.